doc/indices.sgm

```diff
@@ -121,23 +121,32 @@
           The index works by casting all contained smocs to a fixed level, and
           for each pixel at that level, storing which smocs overlap with that
           pixel. This is especially beneficial for "overlaps" queries using
-          the <literal>&amp;&amp;</literal> operator. Two levels of granularity
-          are provided: the default opclass <literal>smoc_gin_ops</literal>
-          works on level 5 with a resolution of 12288 pixels, while the
-          opclass <literal>smoc_gin_ops_fine</literal> works on level 8 with
-          786432 pixels. The downside of that approach is that storing large
+          the <literal>&amp;&amp;</literal> operator.
+          The downside of that approach is that storing large
           smocs like "all sky" (<literal>0/0-11</literal>) produces a large
           number of index entries.
         </para>
+        <para>
+          The default opclass <literal>smoc_gin_ops</literal> defaults to
+          working on level 5 with a resolution of 12288 pixels (12 * 4^5).
+          An alternative granularity can be selected by setting the
+          <literal>order</literal> parameter on the opclass (integer value
+          between 0 and 12; option only available on PG 13 and later).
+          The alternative <literal>smoc_gin_ops_fine</literal> opclass works
+          on level 8 with 786432 pixels.
+        </para>
         <example>
           <title>Index of smoc coverage objects</title>
           <programlisting>
 <![CDATA[CREATE TABLE ivoa (]]>
 <![CDATA[  coverage smoc NOT NULL]]>
 <![CDATA[);]]>
 <![CDATA[-- Put in data now]]>
+<![CDATA[-- Create index with the defaut smoc_gin_ops opclass]]>
 <![CDATA[CREATE INDEX ON ivoa USING GIN (coverage);]]>
 <![CDATA[-- Alternative index with more detail]]>
+<![CDATA[CREATE INDEX ivoa_order_7_idx ON ivoa USING GIN (coverage smoc_gin_ops (order = 7));]]>
+<![CDATA[-- Alternative operator class with fixed order 8]]>
```

Comment:
Not sure, order = **7**, but the text contains 'with fixed order **8**'. Is it correct?

Reply:
It it correct, the 7 belongs to "more detail" on the line before, and order 8 is for the _fine opclass on the next line. But you are right, it is confusing, I'll make it more explicit.